Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM driveclutch/infra-elasticsearch:2.3
 MAINTAINER david@driveclutch
 
-RUN plugin install cloud-aws
+RUN plugin install cloud-aws repository-s3
```

Comment:
INSTALLS: https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3.html

HOW WE CONFIG LATER:  make an s3 bucket and give telematics service roles access to the bucket. Then we do this w/ that bucket: https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-usage.html . Bucket should be only param needed to be fired at cluster at that point.

DR EXERCISE ROUTINE: make a cluster the same way that can access these snapshots as source material. The material will have been synced to ue2/dr-region by regular s3 replication on the s3 bucket this uses